tests/storage_test.go

```diff
@@ -253,5 +253,54 @@ var _ = Describe("Storage", func() {
 				close(done)
 			}, 400)
 		})
+
+		Context("With VM with two PVCs", func() {
+			BeforeEach(func() {
+				// Setup second PVC to use in this context
+				tests.CreatePvISCSI(tests.CustomISCSI, 1, false)
+				tests.CreatePVC(tests.CustomISCSI, false)
+			}, 120)
+
+			AfterEach(func() {
+				tests.DeletePVC(tests.CustomISCSI, false)
+				tests.DeletePV(tests.CustomISCSI, false)
+			}, 120)
+
+			It("should start vm multiple times", func() {
+				checkReadiness()
+
+				vm := tests.NewRandomVMWithPVC(tests.DiskAlpineISCSI)
+				tests.AddPVCDisk(vm, "disk1", "virtio", tests.DiskCustomISCSI)
+				vm.Spec.NodeSelector = map[string]string{"kubernetes.io/hostname": nodeName}
+
+				num := 3
+				By("Starting and stopping the VM number of times")
+				for i := 1; i <= num; i++ {
+					obj := RunVMAndExpectLaunch(vm, false, 120)
+
+					// Verify console on last iteration to verify the VM is still booting properly
+					// after being restarted multiple times
+					if i == num {
+						By("Checking that the VM console has expected output")
+						expecter, _, err := tests.NewConsoleExpecter(virtClient, vm, "serial0", 10*time.Second)
+						defer expecter.Close()
+						Expect(err).To(BeNil())
+						_, err = expecter.ExpectBatch([]expect.Batcher{
+							&expect.BExp{R: "Welcome to Alpine"},
+							&expect.BSnd{S: "root\n"},
+							&expect.BExp{R: "#"},
+							&expect.BSnd{S: "fdisk -l | grep  'Disk /dev/vdb'\n"},
+							&expect.BExp{R: `Disk\s*\/dev\/vdb:\s*`}, //Disk /dev/vdb:
```

Comment:
Low level concern: this path assumes current device naming rules. Can this change in the future?

Reply:
Good point. It can be an issue. However, it is easy to fix. The test is now hard wired for the Alpine image. Once the requirements change or the test distribution changes. I will fix it.